collect_app/src/main/res/xml/preferences.xml

```diff
@@ -75,6 +75,12 @@
             android:key="font_size"
             android:title="@string/font_size" />
         <CheckBoxPreference
+            android:id="@+id/instance_sync"
+            android:defaultValue="true"
+            android:key="instance_sync"
```

Comment:
Even if we do create a new string resource then it would be used only here. Because in classes we use PreferencesKey.java for using the keys